pkg/controller/azdrivernode.go

```diff
@@ -103,12 +103,30 @@ func (r *reconcileAzDriverNode) DeleteAzVolumeAttachments(ctx context.Context, n
 	return nil
 }
 
+func (r *reconcileAzDriverNode) CleanUpAzDriverNodes(ctx context.Context) error {
```

Comment:
Does this need to be a package public function? It seems to only be called within the package.

Reply:
Will do.  I will change the other functions that need to be turned into private as well in next iteration of commit.